clusterloader2/testing/load/config.yaml

```diff
@@ -207,6 +208,15 @@ steps:
       smallJobSize: {{$SMALL_GROUP_SIZE}}
       smallJobsPerNamespace: 1
 
+{{if $ADDITIONAL_PHASES_MODULES}}
```

Comment:
Nit: is this if necessary? Range over nil should be legal operation.

Reply:
I followed what was done in the https://github.com/kubernetes/perf-tests/pull/2483